configure.ac

```diff
@@ -2827,6 +2829,560 @@ if test x$host_win32 = xno; then
+	], [
+		AC_MSG_RESULT(no)
+	])
+
+	AC_MSG_CHECKING(for struct in_pktinfo)
+	AC_TRY_COMPILE([
+		#ifdef HAVE_LINUX_IN_H
+		#include <sys/socket.h>
+		#include <linux/in.h>
+		#else
+		#include <netinet/in.h>
+		#endif
+	], [
+		struct in_pktinfo pktinfo;
+	],[
+		AC_MSG_RESULT(yes)
+		AC_DEFINE(HAVE_IN_PKTINFO, 1, [struct in_pktinfo])
+	], [
+		AC_MSG_RESULT(no)
+	])
+
+	AC_MSG_CHECKING(for struct ip_mreqn)
+	AC_TRY_COMPILE([
+		#if HAVE_LINUX_IN_H
+		#include <sys/socket.h>
+		#include <linux/in.h>
+		#else
+		#include <netinet/in.h>
+		#endif
+	], [
+		struct ip_mreqn mreqn;
+	],[
+		AC_MSG_RESULT(yes)
+		AC_DEFINE(HAVE_IP_MREQN, 1, struct ip_mreqn)
+	], [
+		AC_MSG_RESULT(no)
+	])
+
+	if test x$host_darwin = xno; then
+		AC_MSG_CHECKING(for struct flock64)
+		AC_TRY_COMPILE([
+			#include <fcntl.h>
+		], [
+			struct flock64;
+		],[
+			AC_MSG_RESULT(yes)
+			AC_DEFINE(HAVE_FLOCK64, 1, struct flock64)
+		], [
+			AC_MSG_RESULT(no)
+		])
+	fi
+
+	AC_CHECK_FUNC(lseek64,           [AC_DEFINE(HAVE_LSEEK64, 1, [lseek64])])
+	AC_CHECK_FUNC(mmap64,            [AC_DEFINE(HAVE_MMAP64, 1, [mmap64])])
+	AC_CHECK_FUNC(ftruncate64,       [AC_DEFINE(HAVE_FTRUNCATE64, 1, [ftruncate64])])
+	AC_CHECK_FUNC(posix_fadvise64,   [AC_DEFINE(HAVE_POSIX_FADVISE64, 1, [posix_fadvise64])])
+	AC_CHECK_FUNC(stat64,            [AC_DEFINE(HAVE_STAT64, 1, [stat64])])
+	AC_CHECK_DECL(pipe2,             [AC_DEFINE(HAVE_PIPE2, 1, [pipe2])])
+	AC_CHECK_FUNC(getmntinfo,        [AC_DEFINE(HAVE_GETMNTINFO, 1, [getmntinfo])], [], [[#include <unistd.h>]])
+	AC_CHECK_FUNC(strcpy_s,          [AC_DEFINE(HAVE_STRCPY_S, 1, [strcpy_s])])
+	AC_CHECK_FUNC(strlcpy,           [AC_DEFINE(HAVE_STRLCPY, 1, [strlcpy])])
+	AC_CHECK_FUNC(posix_fadvise,     [AC_DEFINE(HAVE_POSIX_FADVISE, 1, [posix_fadvise])])
+	AC_CHECK_FUNC(ioctl,             [AC_DEFINE(HAVE_IOCTL, 1, [ioctl])])
+	AC_CHECK_FUNC(sched_getaffinity, [AC_DEFINE(HAVE_SCHED_GETAFFINITY, 1, [sched_getaffinity])])
+	AC_CHECK_FUNC(sched_setaffinity, [AC_DEFINE(HAVE_SCHED_SETAFFINITY, 1, [sched_setaffinity])])
+	AC_CHECK_DECL(TIOCGWINSZ,        [AC_DEFINE(HAVE_TIOCGWINSZ, 1, [TIOCGWINSZ])], [], [[#include <sys/ioctl.h>]])
+	AC_CHECK_FUNC(tcgetattr,         [AC_DEFINE(HAVE_TCGETATTR, 1, [tcgetattr])])
+	AC_CHECK_FUNC(tcsetattr,         [AC_DEFINE(HAVE_TCSETATTR, 1, [tcsetattr])])
+	AC_CHECK_DECL(ECHO,              [AC_DEFINE(HAVE_ECHO, 1, [ECHO])], [], [[#include <termios.h>]])
+	AC_CHECK_DECL(ICANON,            [AC_DEFINE(HAVE_ICANON, 1, [ICANON])], [], [[#include <termios.h>]])
+	AC_CHECK_DECL(TCSANOW,           [AC_DEFINE(HAVE_TCSANOW, 1, [TCSANOW])], [], [[#include <termios.h>]])
+
+	AC_CHECK_MEMBER(struct stat.st_birthtimespec, [AC_DEFINE(HAVE_STAT_BIRTHTIME, 1, [struct stat.st_birthtime])], [], [[#include <sys/types.h>], [#include <sys/stat.h>]])
+	AC_CHECK_MEMBER(struct stat.st_atimespec,     [AC_DEFINE(HAVE_STAT_TIMESPEC, 1, [struct stat.st_atimespec])], [], [[#include <sys/types.h>], [#include <sys/stat.h>]])
```

Comment:
Same for AC_CHECK_MEMBERS.

Reply:
We need to control exactly what's the name of the defined variable since it needs to match the corefx one, and `AC_CHECK_MEMBERS` wouldn't allow for it.